src/api/item.js

```diff
@@ -15,6 +15,8 @@ app.get('/', (req, res) => {
 });
 
 app.post('/', (req, res) => {
+    if(!req.body.modelAddress) return res.failureJson("Model address required to make item");
```

Comment:
:+1:

This doesn't check for the validity of the address, but good enough for now!